### DISCUSSION TITLE
按照album的作者搜索，搜索不到album

### DISCUSSION THREAD

**Discussion by heathhou** [hect0x7/JMComic-Crawler-Python#266] [Q&A] [answered]
我使用python代码中的 按照站内搜索（search_site函数）可以正常使用，但是当我按照作者来搜索的时候（search_author函数），搜索出来的album数量为0，试了好多作者，都搜不出来。请问你们有出现过这个问题吗？

**Comment by hect0x7** [comment, marked as answer]
APP端的接口已不支持search_author，只能使用search_site。
网页端仍可以使用search_author

**Reply by heathhou** [reply]
感谢感谢，刚才试了下网页端的，正常了

**Reply by 123hi123** [reply]
想知道怎麼用網頁對用作者查本子的
